README.md

```diff
@@ -22,7 +22,7 @@ and packaging of the application is [Maven](https://maven.apache.org/).
 ### Dependencies
 
 Available dependencies are fetched from Maven Central. Further dependencies (not available in Maven Central) are located in Kitodo/src/main/webapp/WEB-INF/lib.
-The only thing that is not shipped is a Tomcat specific servlet API. This should come along with the Tomcat distribution you are building against.
+Two things that are not shipped are a Tomcat specific servlet API and an ElasticSearch server. First thing should come along with the Tomcat distribution you are building against. Second, you need to download by yourself from [ElasticSearch Download Page](https://www.elastic.co/downloads/past-releases/elasticsearch-2-3-1) and start before Maven build. On the Windows system ElasticSearch starts after open elasticsearch.bat file from bin folder.
```

Comment:
You are linking to version ElasticSearch 2.3.1 but inside Maven dependencies and TravisCI we are using ElasticSearch 5.1.1  and installation guide for 3.x mentioned ElasticSearch 5.x. Should the download link not point to at least ElasticSearch 5.1.1 ?